api/pom.xml

```diff
@@ -59,6 +59,12 @@
 		</dependency>
 				
 		<!-- End OpenMRS core -->
+
+		<dependency>
```

Comment:
Is this really needed?

Reply:
I was looking at what was causing the build errors. I am not seeing these build errors locally.